WcaOnRails/config/locales/fr.yml

```diff
@@ -24,6 +24,8 @@ fr:
     here: "ici"
     wca_id: "ID WCA"
     all_regions: "Toutes les régions"
+    hints:
+      supports_md_html: ""
```

Comment:
@pingskills, thanks for taking care of the French locale :)
You probably noticed that we have spec for i18n now, here is how it works:
- it will report any unused key in any language
- it will report any missing key in English locale

So I think you have to remove the keys you don't use (which you did), but you should probably not add any empty key for the new keys you introduced in the English locale: it's the burden of the maintainer, and spec-wise you're just fine since it will fallback to English and won't raise an error in travis :)
Also adding empty translation for ActiveRecords means French forms would have empty labels and be very weird ^^

For this particular key (`support_md_html`), could you please move the translation you removed from line 409 to here? It's would be the exact same ;)